exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -17,25 +21,24 @@ public class Game {
     int currentPlayer = 0;
     boolean isGettingOutOfPenaltyBox;
     
-    public  Game(){
-    	for (int i = 0; i < 50; i++) {
+    public Game(){
+		addQuestionsToCategories(50);

```

Comment:
Good addition of function for adding the questions. This also has the bonus of allowing other things to go in the game constructor in the future, rather than it being dedicated to adding the questions.

Reply:
Thanks